demos/tests/cases.py

```diff
@@ -73,23 +86,23 @@ def combine_cases(*args):
 def single_option_cases(key, *args):
     return [TestCase(options={} if arg is None else {key: arg}) for arg in args]
 
-def device_cases(*args):
-    return [TestCase(options={opt: device for opt in args}) for device in ALL_DEVICES]
 
 NATIVE_DEMOS = [
-    NativeDemo(subdirectory='crossroad_camera_demo', test_cases=combine_cases(
-        TestCase(options={'-no_show': None,
-            '-i': ImagePatternArg('person-vehicle-bike-detection-crossroad')}),
-        device_cases('-d', '-d_pa', '-d_reid'),
-        TestCase(options={'-m': ModelArg('person-vehicle-bike-detection-crossroad-0078')}),
-        single_option_cases('-m_pa', None, ModelArg('person-attributes-recognition-crossroad-0230')),
-        single_option_cases('-m_reid', None, ModelArg('person-reidentification-retail-0079')),
+    NativeDemo(subdirectory='crossroad_camera_demo',
+                device_keys=['-d', '-d_pa', '-d_reid'],
+                test_cases=combine_cases(
+                    TestCase(options={'-no_show': None,
```

Comment:
Don't change the indentation of cases, please. You've done this correctly with the other demos.